webapp/view/MainPage.view.xml

```diff
@@ -11,16 +11,17 @@
 							<contentLeft>
 								<Image src="images/CernerLogo.png" height="20px" />
 							</contentLeft>
+							<contentMiddle>
+								<Text text="Pacients Assignment"/>
+							</contentMiddle>
 							<contentRight>
 								<Text text="{path: 'systemDate>/dtValue',type: 'sap.ui.model.type.DateTime',formatOptions: {style: 'short'}}" />
+								<Button id="btn1" icon="sap-icon://hint" press="onAbout"/>
```

Comment:
Please use a more meaningful id for the about button.

Reply:
solved